app/main/helpers/validation.py

```diff
@@ -243,6 +255,7 @@ def get_required_fields(self):
 class SharedValidator(DOSValidator):
     # From DOS2 and G8 onwards, validate DUNS number length
     number_string_fields = [('dunsNumber', 9)]
+    word_limit = 500
```

Comment:
Is there no way we can pull this value from the question?

Reply:
Yes, probably. However I think a catch-all limit for all questions would be useful, in case people forget to add a specific limit to future free-text questions.

Reply:
The validator now checks whether `max_length_in_words` is available.